pxtlib/emitter/cloud.ts

```diff
@@ -214,6 +214,8 @@ namespace pxt.Cloud {
     export function isOnline() {
         if (typeof navigator !== "undefined" && isNavigatorOnline()) {
             _isOnline = true;
+        } else {
```

Comment:
oh hm maybe just a local test thing -- it wasn't being toggled to false when i was using chrome devtools to turn off network, but I can remove this

Reply:
Right, it will only toggle to false after the first network call failed with 'disconnected' 0 response / doesn't trust navigator.isOnline response enough to aggressively fail things

Probably fine as nowadays that api is widely supported (though we could just simplify it a bit and `return navigator === "undefined" ? _isOnline : isNavigatorOnline()`) but if it was just from local testing maybe better to leave as is? Will still end up with the same "you're offline" error in either case, and if it actually is offline it won't take any time at all to error out~

Reply:
nicenice i will revert